sourcemash/api/feeds.py

```diff
@@ -10,6 +10,13 @@
 
 from sourcemash.models import Feed, UserItem, Item
 
+import os
+from rq import Queue
+from worker import create_worker
+from worker.scraper import scrape_articles_only
+
+conn = create_worker(os.environ.get("APP_CONFIG_FILE") or "development")
```

Comment:
I think it would make sense to put in sourcemash's config if possible. It's odd to need os and "APP_CONFIG_FILE" in an api file.